api/Dockerfile-ssi-bridge

```diff
@@ -4,7 +4,9 @@ WORKDIR /usr/src/app
 COPY . .
 
 RUN cd shared-modules && npm install && npm run build
-# TODO adjust
+
+WORKDIR /usr/src/app/ssi-bridge
```

Comment:
The best practice require to use always the same directory so /usr/src/app is ok for both services

Reply:
But then I am not able to call `RUN npm install` & `RUN npm run build` how would you solve this?